src/plugins/dashboard/public/dashboard_container/dashboard_container_renderer.tsx

```diff
@@ -107,8 +107,13 @@ export const DashboardContainerRenderer = ({
     { 'dashboardViewport--screenshotMode': isScreenshotMode() },
     { 'dashboardViewport--loading': loading }
   );
+
+  const viewportStyles = css`
+    ${useEuiOverflowScroll('y', true)}
+  `;
```

Comment:
Nice use of `useEuiOverflowScroll` 💯 . 

Nit. The overflow creates a blurry gradient on the viewport that maybe shouldn't appear when the viewport is at the top of the page. Would increasing the padding on the viewport fix this?

<img width="1115" alt="Screenshot 2023-02-06 at 11 05 57 AM" src="https://user-images.githubusercontent.com/1638483/217028628-5f38485c-3577-4e19-902e-6aca75c9e6d8.png">

Reply:
Is adding padding the best solution here? Ideally we wouldn't want to change the look of the dashboard like that. Plus, the gutter created by the tops of the Controls is roughly the same as the gutter between the elements of the Unified Search bar. @andreadelrio, thoughts on this?

Maybe there's some way to remove the fade

Reply:
I can just remove the fade by not passing `true` as the 2nd parameter to `useEuiOverflowScroll`, but it looks a little odd when you're scrolled in the middle of the page without the blur.

I think adding 4px top padding to the controls is sufficient to offset the blur, but if we don't want the additional height added, I can remove the blur.

Here are some screenshots to compare:

### Extra top padding 
#### without adding top padding
<img width="1100" alt="Screenshot 2023-02-06 at 1 26 19 PM" src="https://user-images.githubusercontent.com/1697105/217091812-946fa7ed-2ca1-4555-8d85-bf6ac560a94e.png">

#### with 4px top padding
<img width="1097" alt="Screenshot 2023-02-06 at 1 21 52 PM" src="https://user-images.githubusercontent.com/1697105/217091892-0d603056-a599-4033-98e5-7cc1bc74daec.png">


### Blur vs no blur
#### with blur, scroll midway down the dashboard
 
<img width="1100" alt="Screenshot 2023-02-06 at 1 30 36 PM" src="https://user-images.githubusercontent.com/1697105/217092478-f975272b-d6c6-491b-89de-3e0b2444cb5c.png">

#### no blur, scroll midway down the dashboard
 
<img width="1099" alt="Screenshot 2023-02-06 at 1 28 54 PM" src="https://user-images.githubusercontent.com/1697105/217092171-1295e2a0-5d97-4851-8222-053d0d986fa3.png">

Reply:
Crazy idea, but could we add a scroll listener that removes the blur **only** when at the top? Similar to the scroll listener that was added for options list suggestions.

Reply:
> Is adding padding the best solution here? Ideally we wouldn't want to change the look of the dashboard like that.

Padding the top and bottom is [recommended by EUI](https://elastic.github.io/eui/#/utilities/scroll#vertical-scroll-y). But we could also just remove the mask as Catherine suggested. I don't feel strongly either way.

Reply:
That works for me! I added the bottom border below the editor toolbar too in edit mode too.

#### View mode

<img width="1258" alt="Screenshot 2023-02-08 at 11 49 01 AM" src="https://user-images.githubusercontent.com/1697105/217636201-366ea54e-58bb-4b3a-8dff-a884cd1e4e8e.png">

#### Edit mode

<img width="1260" alt="Screenshot 2023-02-08 at 11 48 50 AM" src="https://user-images.githubusercontent.com/1697105/217636322-08e1b175-712a-44a7-abd8-0f0b592f999e.png">

Reply:
I think that the border doesn't work too well when the page is at the top. In view mode, the border when the page is at the top makes the Controls look very separate from the filter context, which I suppose is somewhat true.

In edit mode, having two borders like that in quick succession is also strange, maybe we leave the border in place, but we don't put the border under the unified search bar in edit mode, and only show the one under the editor?

Reply:
@ThomThomson you mean like this?

<img width="1203" alt="Screenshot 2023-02-08 at 12 21 51 PM" src="https://user-images.githubusercontent.com/1697105/217642551-c618ff89-a5bf-4417-b179-9c6398f76c27.png">

Reply:
Yeah I would say that looks better. The whole pinned section gets one border at the bottom! What do you think?

Reply:
Yeah the extra lines do look pretty busy, but without the border between the two bars, the spacing looks a little tall compared to the other padding on the page, so I think I'll also reduce some padding like this:

<img width="875" alt="Screenshot 2023-02-08 at 1 08 00 PM" src="https://user-images.githubusercontent.com/1697105/217651155-d0e3139f-4be2-4228-b495-043906bcd1b7.png">

Reply:
I like @ThomThomson's suggestion here for the single border in edit mode. Looks like you've already implemented it, so thumbs up from me!